Description: Disable rabagctl.test (which requires root)
Author: Stephen Kitt <skitt@debian.org>

--- a/meson.build
+++ b/meson.build
@@ -551,8 +551,8 @@
 
 env_test = environment()
 env_test.set('LIBRATBAG_DATA_DIR', libratbag_data_dir_devel)
-ratbagctl_test = find_program(join_paths(meson.build_root(), 'ratbagctl.test'))
-test('ratbagctl-test', ratbagctl_test, args: ['-v'], env : env_test)
+#ratbagctl_test = find_program(join_paths(meson.build_root(), 'ratbagctl.test'))
+#test('ratbagctl-test', ratbagctl_test, args: ['-v'], env : env_test)
 
 # ratbag-command uses Swig bindings to call libratbag directly
 swig = find_program('swig')
