Description: Disable valgrind
Author: Stephen Kitt <skitt@debian.org>

--- a/meson.build
+++ b/meson.build
@@ -355,17 +355,6 @@
 	test('test-util', test_util)
 	test('test-iconv-helper', test_iconv_helper)
 
-	valgrind = find_program('valgrind')
-	valgrind_suppressions_file = join_paths(meson.source_root(), 'test', 'valgrind.suppressions')
-	add_test_setup('valgrind',
-		       exe_wrapper : [
-			       valgrind,
-			       '--leak-check=full',
-			       '--quiet',
-			       '--error-exitcode=3',
-			       '--suppressions=' + valgrind_suppressions_file ],
-		       timeout_multiplier: 5)
-
 	executable('test-build-cxx',
 		   ['test/build-cxx.cc'],
 		   dependencies : [ dep_libratbag ],
